File: Ch1-Fundamentals/0016-Countdown/main.py
def countdown_array( num ):
    return list( range(num, -1, -1 ))

result = countdown_array(5)
print(result)
print("Length of the array:", len(result))

'''
Description
This function accepts a number `num` as input, 
    creates a list using `range()` starting from `num` 
    down to 0 with a step of -1, and then returns the 
    list. The length of the array would be `num + 1`, as 
    it includes the numbers from `num` down to 0.
'''

File: Ch1-Fundamentals/0016-Countdown/main.js
function countdownArray( num ) {
    let resultingArr = [];
    for (let i = num; i >= 0; i--) {
        resultingArr.push(i);
    }
    return resultingArr;
}

let result = countdownArray(5);
console.log(result);
console.log("Length of the array:", result.length);

let result2 = countdownArray(10);
console.log(result2);
console.log("Length of the array:", result2.length);